### ISSUE TITLE
scipy.spatial.Voronoi not working with high number coords

### ISSUE THREAD

**Issue by MattReimer** [scipy/scipy#6918]
A little context: We're trying to use Voronoi diagranms with points in GIS shapefiles, which have coordinates with large numbers like `[470121.0, 5118838.0999999996]` 

When I run `scipy.spatial.Voronoi()` on raw points I get:
![figure_1](https://cloud.githubusercontent.com/assets/1063391/21623463/5922aa54-d1b7-11e6-8be1-2a4bed665584.png)

And this is is weird to me. There are shapes but they are really sparse. 

If I shift everything closer to the origin and retry I get:
![figure_2](https://cloud.githubusercontent.com/assets/1063391/21623475/6611599a-d1b7-11e6-8052-170861748c86.png)

That second case is what I expect. This is my first time using this library so I'm not 100% sure I'm not at fault here.

Is this known behavior? Should I just transform my points every time I want to use this?

**Comment by pv** [comment]
Yes, you must center the data around zero first, qhull does not do this for
you.

3.1.2017 23.26 "Matt" <notifications@github.com> kirjoitti:

**Comment by perimosocordiae** [comment]
Looks like the docs don't mention this anywhere (that I can find). Maybe we should add a note?

**Comment by rgommers** [comment]
or do the centering before passing the data to qhull?

**Comment by MattReimer** [comment]
This is good to know. I plan to just transform everything then transform it back when I need it. Would be nice to have that baked in but I'm not sure how much effort that would be.

**Comment by mjmdavis** [comment]
Could the root cause of this be that scipy is using a 32 bit build of QHull?

The [QHull download page](http://www.qhull.org/download/) mentions:
> Includes 32-bit executables, documentation, and sources files. It runs in a command window. Qhull may be compiled for 64-bits.

**Comment by pv** [comment]
Integer size shouldn't be relevant.
